shell/platform/darwin/ios/framework/Source/FlutterEmbedderKeyResponder.mm

```diff
@@ -356,15 +354,15 @@ @interface FlutterEmbedderKeyResponder ()
  *
  * Set by the initializer.
  */
-@property(nonatomic, copy, readonly) FlutterSendKeyEvent sendEvent;
+@property(nonatomic, readonly) FlutterSendKeyEvent sendEvent;
 
 /**
  * A map of pressed keys.
  *
  * The keys of the dictionary are physical keys, while the values are the logical keys
  * of the key down event.
  */
-@property(nonatomic, retain, readonly) NSMutableDictionary<NSNumber*, NSNumber*>* pressingRecords;
+@property(nonatomic, readonly) NSMutableDictionary<NSNumber*, NSNumber*>* pressingRecords;
```

Comment:
These make me a bit nervous, but I guess they are internal-only so hopefully not copying is okay.

Reply:
nit: might as well just be explicit here. Can either keep this `retain` or use `strong instead.

Reply:
The style guide isn't explicit about this, but all the examples leave off `strong` since it's the default:

[For example](https://google.github.io/styleguide/objcguide.html#example)
```
/** The retained Bar. */
@property(nonatomic) Bar *bar;

/** The current drawing attributes. */
@property(nonatomic, copy) NSDictionary<NSString *, NSNumber *> *attributes;
```

@stuartmorgan would you like me to make this a `copy` and see if anything Bad happens (probably nothing).

Reply:
If you're willing to test or audit that it's safe, I would feel better with it as a copy. But it's not changing behavior and this isn't super dangerous like the unretained thing, so if you would prefer to make this behavior-neutral I'm fine with that as well.

Reply:
> /** The retained Bar. */
> @property(nonatomic) Bar *bar;

looks like it's just me being used to always writing `strong` 🤷